base_classes/nyaml/NXparameters.yaml

```diff
@@ -3,33 +3,23 @@ doc: |
   Container for parameters, usually used in processing or analysis.
 type: group
 NXparameters(NXobject):
-  term(NX_CHAR):
+  TERM:
```

Comment:
The more I think about it, the less I like the inconsistence that is in the current code base of not having NX_CHAR either everywhere mentioned or everywhere where the concept is of NX_CHAR removed.
Would also help beginners with telling apart with telling apart concepts from reserved keywords until  https://github.com/FAIRmat-NFDI/nyaml/issues/52 has been implemented